### PR TITLE
Add `osu! Philippines Quickfire Showdown` wiki article

### PR DESCRIPTION
**Pull request by topecnz**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/oPQS/1/en.md [hunk 1]
```diff
@@ -0,0 +1,371 @@
...
+  - oPQS
+  - oPQS2022
+  - oPQS 2022
+---
+
+# osu! Philippines Quickfire Showdown
+
+![oPQS banner](img/banner.png)
+
+The **osu! Philippines Quickfire Showdown** (***oPQS***) is a 1v1 single-elimination open tournament held on both days for those present at the venue. As the name implies, there will be no warmups, and the mappools will be revealed on the spot. The osu! Philippines Quickfire Showdown has become the first ever osu! LAN tournament in the Philippines.
```
Comment (fajar13k) [inline wiki/Tournaments/oPQS/1/en.md:12]:
```suggestion
The **osu! Philippines Quickfire Showdown** (***oPQS***) was a 1v1 single-elimination open tournament held on both days for those present at the venue. As the name implies, there will be no warmups, and the mappools will be revealed on the spot. The osu! Philippines Quickfire Showdown has become the first ever osu! LAN tournament in the Philippines.
```

File: wiki/Tournaments/oPQS/1/en.md [hunk 2]
```diff
@@ -0,0 +1,371 @@
...
+| Streamer | ::{ flag=PH }:: [-Ichinose](https://osu.ppy.sh/users/12635292) |
+| On-site commentator | ::{ flag=PH }:: [Nathos](https://osu.ppy.sh/users/5432869), ::{ flag=PH }:: [BronyPH](https://osu.ppy.sh/users/1492995), ::{ flag=PH }:: [Faltri](https://osu.ppy.sh/users/12372097) |
+| Off-site commentator | ::{ flag=PH }:: [xidorn](https://osu.ppy.sh/users/7904667), ::{ flag=KR }:: [ToGlette](https://osu.ppy.sh/users/1076236) |
+| Referee | ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927), ::{ flag=PH }:: [-Ichinose](https://osu.ppy.sh/users/12635292), ::{ flag=PH }:: [-Pekora-](https://osu.ppy.sh/users/17959882), ::{ flag=PH }:: [omaru1](https://osu.ppy.sh/users/18480113), ::{ flag=PH }:: [ShizuVoice](https://osu.ppy.sh/users/9570552) |
+| Wiki writer | ::{ flag=PH }:: [topecnz](https://osu.ppy.sh/users/2103927), ::{ flag=PH }:: [Afireee](https://osu.ppy.sh/users/7919913) |
+
+## Links
+
+- [News Article](https://osu.ppy.sh/home/news/2022-07-21-osu-philippines-conquest-ph-2022)
```
Comment (fajar13k) [inline wiki/Tournaments/oPQS/1/en.md:68]:
```suggestion
- [News article](https://osu.ppy.sh/home/news/2022-07-21-osu-philippines-conquest-ph-2022)
```

File: wiki/Tournaments/oPQS/1/en.md [hunk 3]
```diff
@@ -0,0 +1,371 @@
...
+- [Challonge (Day 1)](https://challonge.com/oPQS_Day1)
+- [Challonge (Day 2)](https://challonge.com/oPQS_Day2)
+- [Facebook Group](https://www.facebook.com/groups/officialosuphilippines)
```
Comment (fajar13k) [inline wiki/Tournaments/oPQS/1/en.md:71]:
```suggestion
- [Facebook group](https://www.facebook.com/groups/officialosuphilippines)
```

File: wiki/Tournaments/oPQS/1/en.md [hunk 4]
```diff
@@ -0,0 +1,371 @@
...
+- [Livestream](https://www.twitch.tv/osuliveph)
+
+## Participants
+
+- Day 1
+
+  - [-Hanamaru-](https://osu.ppy.sh/users/11348518)
+  - [[Vik]](https://osu.ppy.sh/users/5428448)
```
Comment (fajar13k) [inline wiki/Tournaments/oPQS/1/en.md:79]:
```suggestion
  - [\[Vik\]](https://osu.ppy.sh/users/5428448)
```

Comment (fajar13k) [reply wiki/Tournaments/oPQS/1/en.md:79]: Also apply for other [Vik]

**Review by fajar13k — COMMENTED** [review]
Minor wordings/escape from me

**Review by TPGPL — APPROVED** [review]

**Pull request merged**